src/resonanceReconstruction.hpp

```diff
@@ -36,11 +36,11 @@ using InvElectronVolts = InvElectronVolt;
 constexpr Quantity< RootBarn > rootBarn = 1.0E-12 * centi(meter);
 constexpr Quantity< RootBarn > rootBarns = rootBarn;
 
+namespace ENDF = ENDFtk::resonanceParameters;
+
```

Comment:
I like having namespace shortcuts like this. In my observation, the best-regarded libraries have a "shorthand" feel to them, giving a high content-to-verbosity ratio. Indeed that's true of the C++ language itself - think, for instance, "int" rather than e.g. "Integer".

Reply:
Yes, I do that often with type aliases as well to shorten the name.